samples/generated_samples/snippet_metadata_google.cloud.bigquery.storage.v1.json

```diff
@@ -8,7 +8,7 @@
     ],
     "language": "PYTHON",
     "name": "google-cloud-bigquery-storage",
-    "version": "2.27.0"
```

Comment:
@parthea Is it OK for these to go backward?

Reply:
Yes, these will be updated in the release PR